io.itch.domestique_baston.domestique_baston.yml

```diff
@@ -0,0 +1,32 @@
+app-id: io.itch.domestique_baston.domestique_baston
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: domestique_baston
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --socket=pulseaudio
+modules:
+  - name: domestique_baston
+    buildsystem: simple
+    build-commands:
+      - install -D DOMESTIQUE_BASTON_gles3.x86_64 /app/bin/domestique_baston
+      - install -D -m 0644 *.pdf /app/bin
```

Comment:
not sure what you expect the user to be able to do with these PDF files.

Reply:
It's documentation. I can remove it, or put it somewhere else, but where?

Reply:
they are not accessible, so unless you have a PDF viewer they are doing nothing.